### PR TITLE
Samila Command Line Interface (CLI)

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### What does this implement/fix? Explain your changes.
Command Line Interface (CLI) added.

### CODE REVIEW

File: samila/__main__.py [hunk 1]
```diff
@@ -1,11 +1,105 @@
...
+        type=str,
+        choices=[x.value for x in Marker])
+
+    parser.add_argument('--save-image', help='save image', type=str)
+    parser.add_argument('--depth', help='depth', type=float)
+    parser.add_argument('--save-data', help='save data', type=str)
+    parser.add_argument('--save-config', help='save config', type=str)
+    args = parser.parse_args()
+
+    if args.version:
```
Comment (sepandhaghighi) [inline samila/__main__.py:63]: `--version` flag should only return version number.
```shell
> samila --version
1.3
```

Add another flag like `--info` for this purpose.

File: samila/__main__.py [hunk 2]
```diff
@@ -1,11 +1,105 @@
...
+            cmap=args.cmap,
+            spot_size=args.spot_size,
+            size=args.size,
+            projection=args.projection,
+            marker=args.marker,
+            alpha=args.alpha,
+            linewidth=args.linewidth,
+            rotation=args.rotation,
+        )
+        plt.show()
```
Comment (sepandhaghighi) [inline samila/__main__.py:94]: Add a flag like `--show-image` to control this option.
Default: `True`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 💯 
Please review my comments and remember to update `README.md` as well.

**Comment by sadrasabouri** [comment]
@sepandhaghighi would you please re-review this?
Also, I don't know why CLI takes so long on Windows to run.
Can you check that manually?

**Comment by sadrasabouri** [comment]
@sepandhaghighi I fixed the windows issue by adding the flag `--no-display`. Re-review it once you get a chance.

**Review by sepandhaghighi — APPROVED** [review]
🔥

**Pull request merged**